modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -114,5 +149,9 @@ def birth_date(who)
     def header_field_as_string(key)
       auth_headers&.dig(key).to_s.strip
     end
+
+    def form_data_present?
+      proc { |a| a.form_data.present? }
```

Comment:
What would be a situation where form data would not be present? It seems like we're "not validating in the case of form data not being present." Is that an acceptable edge case?

Reply:
I was wondering the same thing. It is leftover from the original code. HLR has the same (I moved it, but it's the same). I don't know the answer to this question. I will see what I can find out!

Reply:
I checked with Charley and he can't recall exactly why this check was put into place, other than protecting against unwanted stores to the database. Given that the check is simply blocking the validations themselves, I am going to remove it, since the validations will stop the process if data isn't there. Please confirm I'm considering this correctly. The remaining concern: is there a reason we'd want to catch an empty form before validation? Performance is the only thing I can think of, and it doesn't seem like a valid issue.

Reply:
Duh. It's the check to be sure the validations don't run after the removal of PII. So, I don't need both checks, having added `on create`, and this, so this will stay.